### DISCUSSION TITLE
百度云盘的令牌无法获取了

### DISCUSSION THREAD

**Discussion by zhuchaojiebao** [AlistGo/alist#3467] [Q&A] [answered]
百度网盘的刷新令牌后提示错误

**Comment by youtehub** [comment]
你网络的问题？这边都获取到了唉

**Reply by zhuchaojiebao** [reply]
我在国内，换了好多节点，也换了好几个vpn  全局模式也开了。就是获取不到。头大。

**Reply by DaiJiequn** [reply]
这玩意不要vpn吧，我国内网络正常访问

**Reply by zhuchaojiebao** [reply]
> 这个玩意不要vpn吧，我国内网络正常访问

我也不清楚，反正我这边一直无法获取到百度网盘的令牌。点击获取刷新令牌后就是提升 你遇到了错误 
错误代码：redirect_uri_mismatch
错误信息：无效的重定向uri
详细描述：授权返回调整页面地址错误，具体内容请开发者参考“redirect uri”验证方式

**Reply by 3028356499** [reply]
怎么解决呢,我也是相同的问题

**Comment by goodrabbit** [comment]
> 百度网盘的刷新令牌后提示错误

同样的问题+1

**Comment by goodrabbit** [comment]
貌似作者更改了API的回调地址导致的。研究了一下，正确获得令牌的链接应该是这个：

https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

![屏幕截图 2023-02-21 182552](https://user-images.githubusercontent.com/5088561/220319077-fe3b9bf5-a8da-4cdf-9598-dbea6d1f3cfa.jpg)

**Reply by 3028356499** [reply]
<img width="237" alt="image" src="https://user-images.githubusercontent.com/119486005/223024470-4cc8b93b-644f-4fea-afb9-c6fa7e2fc5a6.png">

**Reply by 3028356499** [reply]
他这个一直加载不了

**Reply by zhuchaojiebao** [reply, marked as answer]
> https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

**Reply by wgxsun** [reply]
好像你这个是对的，就人一命，受我一拜！

**Reply by AndrewZion** [reply]
大佬牛逼

**Reply by goldmakers** [reply]
大佬太厉害了

**Reply by SummerDays18** [reply]
义父！

**Reply by lynjupt** [reply]
大佬给力

**Comment by Kayxue** [comment]
今天好像又趴了？
![image](https://github.com/alist-org/alist/assets/26020165/44021082-1936-4da5-96d8-40bf7c8cd6d1)

**Comment by xiao-k233** [comment]
我这里一直提示不行，明明登陆了说我没登陆
![](https://s2.loli.net/2023/12/24/sb9QmtR1FGonuLX.png)
![](https://s2.loli.net/2023/12/24/dJPyOBCk8YxAMXQ.png)
